cardano-db/src/Cardano/Db/Schema/Types.hs

```diff
@@ -8,6 +8,8 @@ module Cardano.Db.Schema.Types
   , PaymentAddrHash (..)
   ) where
 
+import           Cardano.Prelude
+
```

Comment:
What benefit does this actually give us?

Reply:
It's a standard prelude we are supposed to use? I guess I don't have to talk about the benefits of a custom Prelude?

Reply:
> It's a standard prelude we are supposed to use? 

No, there is no mandate.